stack/services/src/main/java/org/apache/usergrid/services/notifications/impl/ApplicationQueueManagerImpl.java

```diff
@@ -175,13 +179,13 @@ public void queueNotification(final Notification notification, final JobExecutio
             //process up to 10 concurrently
             Observable o = rx.Observable.create( new IteratorObservable<Entity>( iterator ) )
 
-                                        .flatMap( entity -> Observable.just( entity ).map( entityListFunct )
-                                                                      .doOnError( throwable -> {
-                                                                          LOG.error( "Failed while writing",
-                                                                              throwable );
-                                                                      } ).subscribeOn( Schedulers.io() )
+                                        .flatMap(entity -> Observable.just(entity).map(entityListFunct)
+                                            .doOnError(throwable -> {
+                                                LOG.error("Failed while writing",
+                                                    throwable);
+                                            }).subscribeOn(Schedulers.io())
```

Comment:
Since we block on line 190, we won't need to subscribe on Schedulers.io().  I know you didn't add this, but can we remove it since we're spinning up another thread that we don't need?